packages/data-explorer/src/router.ts

```diff
@@ -1,34 +1,26 @@
-import Vue from 'vue'
-import Router from 'vue-router'
 import MainView from './views/MainView.vue'
 
-Vue.use(Router)
-
-const packageJson = require('../package.json')
-
-const router = new Router({
-  base: process.env.NODE_ENV === 'production' ? packageJson.name : process.env.BASE_URL,
-  routes: [
-    {
-      path: '/:entity/:view',
-      component: MainView,
-      name: 'main-view'
-    },
-    { path: '/:entity?',
-      redirect: to => {
-        let entity = 'sys_ts_DataExplorerEntitySettings'
-        if (to.params.entity) {
-          entity = to.params.entity
-        }
+const routes = [
```

Comment:
should the file name be changed to routes ?

Reply:
That makes sense indeed. Changed